src/python/src/grpc/_adapter/rear.py

```diff
@@ -41,7 +41,7 @@
 from grpc.framework.foundation import activated
 from grpc.framework.foundation import logging_pool
 
-_THREAD_POOL_SIZE = 100
```

Comment:
This constant is unused; please delete it (and sorry for the red herring in today's investigation!).